src/DiscreteFilter.cpp

```diff
@@ -0,0 +1,314 @@
+/*
+ * DiscreteFilter.cpp
+ *
+ *  Created on: 19/09/2014
+ *      Author: rafaelsaback
+ */
+
+#include "DiscreteFilter.hpp"
+
+namespace base {
```

Comment:
+1 for changing the namespace
and backward compatibility for the already existing